### PR TITLE
Improve Exclusion handling backups

### PR DESCRIPTION
**Pull request by youradds**
Tweaked the exclusion rules regex so that *:public_html would work the same as domain.com:public_html (but for all domains)

### CODE REVIEW

File: bin/v-backup-user [hunk 1]
```diff
@@ -172,19 +172,16 @@ start_time=$(date '+%s')
...
 # Validate available disk space (take usage * 2, due to the backup handling)
 let u_disk=$(($(get_user_disk_usage) * 2))
 let v_disk=$(($(stat -f --format="%a*%S" $BACKUP)))/1024/1024
 
 if [ "$u_disk" -gt "$v_disk" ]; then
-	let u_disk_original=$(get_user_disk_usage)
 	# Always notify on failure
-	echo "Not enough disk space available ($v_disk mb) to perform the backup  of $user. ( $u_disk_original mb * 2 = $u_disk mb). https://hestiacp.com/docs/server-administration/backup-restore.html" | $SENDMAIL -s "$subj" "$email" "yes"
-	# Deleting task from queue
-	sed -i "/v-backup-user $user /d" $HESTIA/data/queue/backup.pipe
```
Comment (jaapmarcus) [inline bin/v-backup-user]: We should not undo this line other wise it keeps retrying every time..

File: bin/v-backup-user [hunk 2]
```diff
@@ -195,38 +192,36 @@ fi
 tmpdir=$(mktemp -p $BACKUP_TEMP -d)
 
 if [ "$?" -ne 0 ]; then
-	echo "Can't create tmp dir $tmpdir" | $SENDMAIL -s "$subj" "$email" "yes"
-	# Deleting task from queue
-	sed -i "/v-backup-user $user /d" $HESTIA/data/queue/backup.pipe
```
Comment (jaapmarcus) [inline bin/v-backup-user]: Same here

Comment (youradds) [reply bin/v-backup-user]: I'm not sure where these changes came from? All I did was fork from the main repo, and change:

"$domain"

to

"$domain\|*"

And:

^$domain$

to

^$domain\|\*$

etc

So not sure where any of that came from? (I'm still trying to get my head around the git stuff =))

File: bin/v-backup-user (hunk 1 above)
Comment (youradds) [reply bin/v-backup-user]: How / can I delete the commits? I don't seem to have a revoke/ cancel commit? I have tweaked the files and it should be good now - but trying to add it on top still seems to be using the other commits as well! This file should be good now (just not commited);

https://github.com/youradds/hestiacp/blob/main/bin/v-backup-user

**Pull request closed without merging**